doc/version_history.rst

```diff
@@ -6,6 +6,20 @@
 Version History
 ###############
 
+v0.21.2
+-------
+
+* `HexapodCommander`: make compatible with ts_xml 10.1 (while retaining backwards compatibility).
+
+Requires:
+
+* ts_hexapod_controller 1.2.0
+* ts_hexrotcomm 0.23
+* ts_salobj 6.3
+* ts_idl 3.4
+* ts_xml 7.1
```

Comment:
`7.1` -> `10.1`

Reply:
This makes the package compatible with 10.1 *and* earlier versions.

Reply:
Thanks for the clarification.